tests/core/test_graph.py

```diff
@@ -1864,7 +1864,11 @@ def test_node_degrees(use_ilocs):
             node_iloc = g.node_ids_to_ilocs([node_id])[0]
             assert expected[node_id] == degrees[node_iloc]
     else:
-        assert expected == degrees
+        for node_id in g.nodes():
+            if node_id in expected.keys():
```

Comment:
This would be better done by adding the isolated node to `expected` (`expected = {..., 6: 0}`).